aseba/launcher/src/launcher.cpp

```diff
@@ -200,6 +201,26 @@ bool Launcher::openUrl(const QUrl& url) {
     return true;
 }
 
+/* **************
+* it writes local application information to application settings 
+* for being available in next execution - 
+* the function must be called in the program exit and not all times  
+************** */
+void Launcher::writeSettings(){
+   // the use of local browser available
+    settings.setValue("mainwindow/useLocalBrowser",QVariant(useLocalBrowser) );
+
```

Comment:
Do not use slashes ('/' and '\') in section or key names; the backslash character is used to separate sub keys (see below). On windows '\' are converted by QSettings to '/', which makes them identical.